projects/aca-shared/testing/pages/page.ts

```diff
@@ -26,7 +26,12 @@
 import { browser, by, ElementFinder } from 'protractor';
 import { Logger } from '@alfresco/adf-testing';
 import { USE_HASH_STRATEGY } from './../configs';
-import { Utils, waitElement, waitForPresence, waitForVisibility } from '../utilities/utils';
+import {
+  Utils,
+  waitElement,
+  waitForPresence,
```

Comment:
Those wait sentence needs to be removed and substitute with the one on Browser action adf-testing

Reply:
ACA testing framework refactoring can be part of a different ticket, since it will add more changes for review.